### DISCUSSION TITLE
About left/right 'windows' keys (LSUPER/RSUPER) ...reserved?

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1117] [Q&A] [answered]
I noticed that if you go into GUI->Miscellaneous and try to map left or right windows keys, the popup selector displays 'LGUI' & 'RGUI' when these keys are pressed, and neither key is considered a 'valid' hotkey value and cannot be selected/saved.

I'm just curious as to why this is so ~ are these 2 keys excluded from being hotkeys for any particular reason, ie; as I suggest being reserved for some purpose?

TIA

**Comment by midwan** [comment, marked as answer]
Those buttons are considered "modifiers" like Ctrl/Alt/Shift, so they cannot be used by themselves, only in combination with something else.

**Comment by giantclambake** [comment]
Thanks, that clears it up in my head
